cli/command/environment.go

```diff
@@ -211,20 +205,15 @@ func (e *EnvironmentCommand) read(c *cli.Context) error {
 }
 
 func (e *EnvironmentCommand) setScale(c *cli.Context) error {
-	args, err := extractArgs(c.Args(), "ENVIRONMENT_NAME")
+	args, err := extractArgs(c.Args(), "ENVIRONMENT_NAME", "SCALE")
 	if err != nil {
 		return err
 	}
 
 	req := models.UpdateEnvironmentRequest{}
-	if c.IsSet("min-scale") {
-		minScale := c.Int("min-scale")
-		req.MinScale = &minScale
-	}
-
-	if c.IsSet("max-scale") {
-		maxScale := c.Int("max-scale")
-		req.MaxScale = &maxScale
+	scale, err := strconv.Atoi(args["SCALE"])
+	if err == nil {
```

Comment:
This seems a little odd to me, not sure why we wouldn't use standard error checking here?
```go
scale, err := strconv...
if err != nil { 
    return err
}

req := models.UpdateEnvironmentRequest{
    Scale: &scale,
}
...
```

Reply:
That is odd - not sure how that happened 😕 
✅